core/src/test/resources/sql/cast-with-format.iq

```diff
@@ -361,7 +351,7 @@ EXPR$0
 select cast('2017-05-31' as
     timestamp FORMAT 'RRRR-MM-DD');
 EXPR$0
-2017-05-31 00:00:00
+0017-05-31 00:00:00
```

Comment:
why is this correct?

Reply:
Thanks for catching that, I accidentally edited that line instead of one below.. have refactored some of this code.